samples/glean/src/androidTest/java/org/mozilla/samples/glean/GleanTestRunner.kt

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.glean
+
+import androidx.annotation.VisibleForTesting
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.runner.AndroidJUnitRunner
+import okhttp3.mockwebserver.Dispatcher
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okhttp3.mockwebserver.RecordedRequest
+
+/**
+ * Create a mock webserver that accepts all requests.
+ * @return a [MockWebServer] instance
+ */
+private fun getMockWebServer(): MockWebServer {
```

Comment:
I was confused by this name, as the docs say it creates one and the next method has the same signature as well. Care to rename this into `createMockWebServer` or `instantiateMockWebServer` or something more descriptive?